cmd/up/forwards.go

```diff
@@ -129,6 +112,67 @@ func (up *upContext) sshForwards(ctx context.Context) error {
 	return nil
 }
 
+func addToForwarder(up *upContext) error {
+	ticker := time.NewTicker(1 * time.Second)
+	to := time.NewTicker(up.Dev.Timeout.Resources)
```

Comment:
Let's use a 10s timeout. Otherwise, if the port is taken by other app, we would fail very late